content/source/docs/github-actions/common-actions/working-directory.html.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: "github-actions"
+page_title: "Working Directory - Terraform GitHub Actions"
+sidebar_current: "docs-github-actions-common-actions-working-directory"
```

Comment:
Oh argh, I forgot to mention on the last pass of reviews that we don't need sidebar_current at all anymore. 😭 You can leave it in, but there's no need to keep updating it, it's inert now.